wpimath/src/main/native/include/frc/estimator/UnscentedTransform.h

```diff
@@ -47,12 +47,17 @@ SquareRootUnscentedTransform(
                                   const Vectord<CovDim>&)>
         residualFunc,
     const Matrixd<CovDim, CovDim>& squareRootR) {
-  // New mean is usually just the sum of the sigmas * weight:
-  //       n
-  // dot = Σ W[k] Xᵢ[k]
-  //      k=1
+  // New mean is usually just the sum of the sigmas * weights:
+  //      2n
+  //   x̂ = Σ Wᵢ⁽ᵐ⁾𝒳ᵢ
+  //      i=0
+  // equations (19) and (23) show this
+  // but we allow a custom function, usually for angle wrapping
   Vectord<CovDim> x = meanFunc(sigmas, Wm);
 
+  // Form an intermediate matrix S_bar as:
+  //   [√{W₁⁽ᶜ⁾}*(𝒳_{1:2L} - x̂) √{Rᵛ}]
```

Comment:
Newlines around this. Also, I assume this isn't convolution, so the `*` should be either omitted or replaced with `⋅`.